src/Blobstream.sol

```diff
@@ -36,7 +36,12 @@ struct Signature {
 /// block headers.
 /// @dev DO NOT REMOVE INHERITANCE OF THE FOLLOWING CONTRACTS: Initializable, UUPSUpgradeable and
 /// OwnableUpgradeable! They're essential for upgradability.
-contract Blobstream is IDAOracle, Initializable, UUPSUpgradeable, OwnableUpgradeable {
+contract Blobstream is
```

Comment:
let's not format and downgrade the version in the same PR.

Also, which formatter did you use to have all of these changes? ideally, we have that in the CI to enforce a certain formatting.